lib/ansible/plugins/loader.py

```diff
@@ -914,10 +914,16 @@ def all(self, *args, **kwargs):
             basename = os.path.basename(name)
 
             if basename == '__init__' or basename in _PLUGIN_FILTERS[self.package]:
+                # either empty or contains base classes
```

Comment:
```suggestion
                # either empty or ignored by the module blocklist
```
BTW, the blocklist is still called blacklist in https://github.com/ansible/ansible/blob/devel/lib/ansible/plugins/loader.py#L1064=L1086